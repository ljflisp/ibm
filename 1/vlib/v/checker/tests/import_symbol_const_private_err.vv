import v.scanner

fn main() {
  println(scanner.single_quote)
}
