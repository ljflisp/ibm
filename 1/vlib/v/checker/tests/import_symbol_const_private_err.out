vlib/v/checker/tests/import_symbol_const_private_err.vv:4:19: error: constant `v.scanner.single_quote` is private
    2 | 
    3 | fn main() {
    4 |   println(scanner.single_quote)
      |                   ~~~~~~~~~~~~
    5 | }
